lib/cpp/src/thrift/transport/TServerSocket.cpp

```diff
@@ -60,7 +60,7 @@
 #endif // _WIN32
 #endif
 
-#if defined(_WIN32) && (_WIN32_WINNT < 0x0600)
+#if defined(_WIN32) && (_WIN32_WINNT < 0x0600) && !defined(AI_ADDRCONFIG)
   #define AI_ADDRCONFIG 0x0400
```

Comment:
I'm not convinced this needs to be defined here; every windows platform windows supports has winsock2, which includes ws2def.h, which defines this and AI_PASSIVE which is used right alongside AI_ADDRCONFIG in the code... so I am going to remove the #if block here.